tests/configs/tls13-only.h

```diff
@@ -26,6 +26,3 @@
 #undef MBEDTLS_SSL_DTLS_CLIENT_PORT_REUSE
 #undef MBEDTLS_SSL_DTLS_CONNECTION_ID
 #undef MBEDTLS_SSL_DTLS_CONNECTION_ID_COMPAT
-
-/* Enable some invasive tests */
-#define MBEDTLS_TEST_HOOKS
```

Comment:
Not a blocker, but I am curious as to why moving is to a component set parameter was required. All tls12_only tests need to asser it.

Reply:
tls13-only.sh is an mbedtls user config file and MBEDTLS_TEST_HOOKS now belongs to crypto_config.h. That's the purpose of the change, having MBEDTLS_TEST_HOOKS defined in crypto_config.h not mbedtls_config.h.